src/components/balance/PlmBalance.vue

```diff
@@ -235,9 +238,56 @@ export default defineComponent({
     const openFaucetModal = (): void => {
       emit('update:is-open-modal-faucet', true);
     };
+
     const canUnlockVestedTokens = computed(() => props.accountData.vested.gtn(0) && !isH160.value);
 
-    const unlockVestedTokens = async (): Promise<void> => {
+    const handleTransactionError = (e: Error): void => {
+      console.error(e);
+      store.dispatch('general/showAlertMsg', {
+        msg: `Transaction failed with error: ${e.message}`,
+        alertType: 'error',
+      });
+    };
+
+    const handleResult = (result: ISubmittableResult): void => {
+      const status = result.status;
+      if (status.isInBlock) {
+        const msg = `Completed at block hash #${status.asInBlock.toString()}`;
+
+        store.dispatch('general/showAlertMsg', {
+          msg,
+          alertType: 'success',
+        });
+
+        store.commit('general/setLoading', false);
+      } else {
+        if (status.type !== 'Finalized') {
+          store.commit('general/setLoading', true);
+        }
+      }
+    };
+
+    const { callFunc } = useExtrinsicCall({
+      onResult: handleResult,
+      onTransactionError: handleTransactionError,
+    });
+
+    const unlockVestedTokensEthExtrinsic = async (): Promise<void> => {
+      try {
+        const fn: SubmittableExtrinsicFunction<'promise'> | undefined = api?.value?.tx.vesting.vest;
```

Comment:
![Screen Shot 2022-01-20 at 1 05 26 PM](https://user-images.githubusercontent.com/40356749/150335657-8af64c5e-c065-4d4b-9487-15ae2f6fd697.png)

Clicking the unlock button will only unlock the claimable amount instead of the vested amount. Plus, the users will not know how much is claimable or see the vested amount change with their actions.

How about we display the claimable vested amount in real-time?
https://github.com/polkadot-js/apps/blob/5174f5ffbecf583d6b9d47d07c46ae777e0032f9/packages/react-components/src/AddressInfo.tsx#L273

Reply:
@hoonsubin 
Bobo is working on this in another branch (this PR is made for debugging).
@bobo-k2  Can I close this PR since you have merged to your branch already?

Reply:
Yes. I merged the code. You can close this one. I will create PR which shows claimable vested amount soon

Reply:
okay! Great